oauth2_provider/models.py

```diff
@@ -506,7 +506,7 @@ class AbstractIDToken(models.Model):
         null=True,
         related_name="%(app_label)s_%(class)s",
     )
-    token = models.TextField(unique=True)
+    token = models.TextField(unique=True, null=True)
```

Comment:
Do we want unique here anymore? I would have thought this would be null=True and blank=True

Reply:
`blank` just controls how it appears in the GSAdmin I think.. but I can update both!

Reply:
TIL!
https://stackoverflow.com/a/21812150